ios/FluentUI.Demo/FluentUI.Demo/Demos/AvatarViewDemoController.swift

```diff
@@ -10,28 +10,58 @@ class AvatarViewDemoController: DemoController {
     override func viewDidLoad() {
         super.viewDidLoad()
 
-        createSection(withTitle: "Circle style for person", name: "Kat Larrson", image: UIImage(named: "avatar_kat_larsson")!, style: .circle)
-        createSection(withTitle: "Square style for group", name: "NorthWind Traders", image: UIImage(named: "site")!, style: .square)
-        createSection(withTitle: "With image based frame", name: "Kat Larrson", image: UIImage(named: "avatar_kat_larsson")!, style: .circle, withColorfulBorder: true)
+        createSection(withTitle: "Circle style for person",
+                      name: "Kat Larrson",
+                      image: UIImage(named: "avatar_kat_larsson")!,
+                      style: .circle)
+
+        createSection(withTitle: "Square style for group",
+                      name: "NorthWind Traders",
+                      image: UIImage(named: "site")!,
+                      style: .square)
+
+        createSection(withTitle: "With image based frame",
+                      name: "Kat Larrson",
+                      image: UIImage(named: "avatar_kat_larsson")!,
+                      style: .circle,
+                      withColorfulBorder: true)
+
+        createSection(withTitle: "Circle style for person with presence",
```

Comment:
nit: maybe instead of switch to show presence?

Reply:
That's a good idea